m/main_glider_data_processing_dt.m

```diff
@@ -163,7 +171,7 @@
 glider_toolbox_ver = configGliderToolboxVersion();
 
 
-%% Configure external programs paths.
+%% Configure external programs paths. (not used anyway for now)
```

Comment:
Yes,  the WRC programs are used flo Slocum gliders only.
Could you revert the comment, though?